aws/resource_aws_acm_certificate.go

```diff
@@ -216,7 +216,7 @@ func convertValidationOptions(certificate *acm.CertificateDetail) ([]map[string]
 				for _, validationEmail := range o.ValidationEmails {
 					emailValidationResult = append(emailValidationResult, *validationEmail)
 				}
-			} else {
+			} else if *o.ValidationStatus == acm.DomainStatusPendingValidation {
```

Comment:
Found during testing: Right after `RequestCertificate`, ACM API will omit `ValidationStatus` completely from the API response. Adjusting this to also retry if `o.ValidationStatus == nil` helped fix the Go panic.